src/lcm/grids.py

```diff
@@ -30,13 +30,13 @@ class DiscreteGrid(Grid):
 
     """
 
-    options: Collection[int | float]
+    options: type
```

Comment:
Just a note that this should be documented really well, if just for my own benefit. I realised only now and after a couple of loops in my brain that we are talking about passing the type and not an instance!